Magento_Directory/templates/currency.phtml

```diff
@@ -19,15 +19,15 @@
     <ul class="dropdown-list__list dropdown-list__item--parent">
         <li class="dropdown-list__item">
             <span class="dropdown-list__link"
-                  data-dropdown="language-<?php echo $block->escapeHtml($block->getCurrentCurrencyCode()) ?>"
+                  aria-controls="language-<?php echo $block->escapeHtml($block->getCurrentCurrencyCode()) ?>"
             >
                 <?php echo __('Currency') ?>: <?php echo $block->escapeHtml($currentCurrencyCode) ?> - <?php echo @$block->escapeHtml($currencies[$currentCurrencyCode]) ?>
                 <svg class="dropdown-list__icon dropdown-list__icon--inner">
                     <use xlink:href="<?= $block->getViewFileUrl('images/icons-sprite.svg#arrow-down'); ?>"></use>
                 </svg>
             </span>
             <ul class="dropdown-list__inner-list dropdown-list__inner-list--level1"
-                data-dropdown="language-<?php echo $block->escapeHtml($block->getCurrentCurrencyCode()) ?>"
+                aria-controls="language-<?php echo $block->escapeHtml($block->getCurrentCurrencyCode()) ?>"
```

Comment:
We are going to use shorthands so you can change `<?php echo` to `<?=` in whole PR

Reply:
replaced on all theme template